### ISSUE TITLE
Iterate over multiple SQL files from Directory and validate syntax

### ISSUE THREAD

**Issue by shiv24saxena** [Seddryck/NBi#674]
Hi,

I have multiple SQL files (contain create view statement) in a directory, whose syntax I want to validate through NBi. However I am not able to iterate over the files and pass it to my query execution. Below is the snippet of my testsuite.

![image](https://user-images.githubusercontent.com/75632216/150389735-0e7ec999-6fcd-4385-b8c0-2e5b87f56630.png)

**Comment by Seddryck** [comment]
What's the error that you're receiving with this snippet?

**Comment by shiv24saxena** [comment]
Thanks Seddryck for your response. It's not able to resolve file path (see attached screenshot). Files are present in C:/Test directory so in file attribute of query section I should get C:/TEST/DIM_ABC_VIEW.sql for DIM_ABC_VIEW.sql & C:/Test/DIM_ADJ_VIEW.sql for DIM_ADJ_VIEW.sql
![image](https://user-images.githubusercontent.com/75632216/150623685-f08c5432-e949-42a2-8269-051d5ff5a3d6.png)

**Comment by Seddryck** [comment]
Have you tried to use `file="~{@sqlview}"` ?

See doc at http://www.nbi.io/docs/primitive-result-set/#external-definition

**Comment by shiv24saxena** [comment]
Yes, I tried, It throws similar error. Framework is not able to resolve variable in file attribute.
![image](https://user-images.githubusercontent.com/75632216/151408715-43df1593-dc49-4b65-898b-2b73ef573dd4.png)

![image](https://user-images.githubusercontent.com/75632216/151408838-72225743-90c5-4374-9290-120fedae3d3c.png)

**Comment by Seddryck** [comment]
Huuum, i think that the variables are not supported for execution. Something to add in framework.

**Comment by shiv24saxena** [comment]
Thanks for the reply. Is there any way to meet the below requirement with NBi?
"Validate Syntax of multiple view , by reading definition from SQL files (1 file per view) present in a directory"

**Comment by Seddryck** [comment]
Unfortunately not and I'm really busy with a huge migration to the v2.0 of NBi so I won't include this soon. But I'm adding that as a feature of v2.1

**Comment by shiv24saxena** [comment]
Thanks. Eagerly waiting for new major release 2.0, any tentative release date :)
